web/src/js/components/feedback/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+export default class Feedback extends React.Component {
+    constructor() {
+        super()
+        this.reactivate = this.reactivate.bind(this);
+    }
+
+    reactivate(e) {
```

Comment:
Not sure I get entirely what this function is doing? It's sending out an email and reactivating? Perhaps split them in different methods